### DISCUSSION TITLE
How to make werf to print output of init containers?

### DISCUSSION THREAD

**Discussion by vladsf** [werf/werf#5764] [General]
While running 'werf helm upgrade --install .. ' I can not make werf to print output of init containers.
 --log-color-mode=on --log-pretty=true --log-terminal-width=200 --status-progress-period=10 do not work.

Any ideas?

**Comment by ilya-lesikov** [comment]
version?

**Comment by ilya-lesikov** [comment]
Also, you need to specify `--wait` flag for this.

**Comment by vladsf** [comment]
The latest stable version 1.2.xx
